docs/website/docs/getting-started/pytorch.md

```diff
@@ -1,124 +1,5 @@
-
-### Compile the model to MLIR
-
-First, we need to trace and compile our model to MLIR:
-
-```python
-model = # ... the model we're compiling
-example_input = # ... an input to the model with the expected shape and dtype
-mlir = torch_mlir.compile(
-    model,
-    example_input,
-    output_type=torch_mlir.OutputType.LINALG_ON_TENSORS,
-    use_tracing=True)
-```
-
-The full list of available output types can be found
-[here](https://github.com/llvm/torch-mlir/blob/6403c0e56f0e93e231df1c8d3dc78df7dd721b80/python/torch_mlir/__init__.py#L19)
-and includes linalg on tensors, mhlo, and tosa.
-
-### Compile the MLIR to an IREE VM flatbuffer
-
-Next, we compile the resulting MLIR to IREE's deployable file format:
-
-```python
-iree_backend = "llvm-cpu"
-iree_vmfb = iree_torch.compile_to_vmfb(mlir, iree_backend)
-```
-
-Here we have a choice of backend we want to target. See the
-[Deployment Configurations](https://iree-org.github.io/iree/deployment-configurations/)
-section of this site for a full list of targets and configurations.
-
-The generated flatbuffer can now be serialized and stored for another time or
-loaded and executed immediately.
-
-### Load the VM flatbuffer into IREE
-
-Next, we load the flatbuffer into the IREE runtime. `iree_torch` provides a
-convenience method for loading this flatbuffer from Python:
-
-```python
-invoker = iree_torch.load_vmfb(iree_vmfb, iree_backend)
-```
-
-### Execute the model via IREE
-
-Finally, we can execute the loaded model:
-
-```python
-result = invoker.forward(example_input)
-```
-
-## Training
-
-Training with PyTorch in IREE is supported via `functorch`. The steps for
-loading the model into IREE, once defined, are nearly identical to the above
-example.
-
-You can find a full end-to-end example of defining a basic regression model,
-training with it, and running inference on it
-[here](https://github.com/iree-org/iree-torch/blob/main/examples/regression.py).
-
-## Native / On-device Training
-
-A small (~100-250KB), self-contained binary can be built for deploying to
-resource-constrained environments. An example illustrating this can be found in
-[this example](https://github.com/iree-org/iree-torch/tree/main/examples/native_training).
-This binary runs a model without a Python interpreter.
-
-## Samples
-
-| Colab notebooks |  |
-| -- | -- |
-Inference on BERT | [![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/iree-org/iree-torch/blob/main/examples/bert.ipynb)
-
-| Example scripts |
-| -- |
-| [Basic Inference and Training Example](https://github.com/iree-org/iree-torch/blob/main/examples/regression.py) |
-| [Native On-device Training Example](https://github.com/iree-org/iree-torch/tree/main/examples/native_training) |
+PyTorch support is community driven. To import programs from PyTorch and use
+them with IREE, see projects like [SHARK](https://github.com/nod-ai/SHARK) from
+our partners at nod.ai.
```

Comment:
Maybe we should also mention https://github.com/llvm/torch-mlir ? 

Is https://github.com/openxla/iree/tree/main/compiler/src/iree/compiler/InputConversion/TMTensor still relevant?

I haven't been following this side of the project very closely, but it seems like enough pieces exist in MLIR/Torch/IREE to compile and run programs, and SHARK provides more docs/APIs/samples. Still seems good to mention the community aspect and point to examples in SHARK (or other projects) that show e2e workflows, but I think we could elaborate more here.

Reply:
TM Tensor is very important for complex models.